### PR TITLE
added the index method and view skeleton

### PR DESCRIPTION
**Pull request by oscargay**
added the basic method for the equipment index and structured a simple skeleton for the equipment index view

### CODE REVIEW

**Comment by oscargay** [comment]
@Nobaltha

File: app/views/equipment/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1>Equipment Near You</h1>
+
+<% @equipments.each do |equipment| %>
+  <h5> <%= equipment.name  %> <%= "(£#{equipment.price})"  %> <%= "Uploaded by #{equipment.user.first_name}"  %> </h5>
```
Comment (Nobaltha) [inline app/views/equipment/index.html.erb:4]: Why did you use a h5 tag instead of a p or li?

Comment (oscargay) [reply app/views/equipment/index.html.erb:4]: I have no idea, thanks @Nobaltha

Comment (oscargay) [reply app/views/equipment/index.html.erb:4]: Will change now

**Comment by oscargay** [comment]
@Nobaltha changed h5 tag to p tag

**Review by Nobaltha — APPROVED** [review]
Well done 🚀

**Pull request merged**